src/cli/init.go

```diff
@@ -136,3 +148,18 @@ func (c icli) InitCMD() *cli.Command {
 		},
 	}
 }
+
+func loadConfigFileSchemaVersion(resources fs.FS) (string, error) {
+	bytes, err := fs.ReadFile(resources, path.Join("assets", "versions", "versions.json"))
+	if err != nil {
+		return "", err
+	}
+
+	versionConfig := make(map[string]string)
```

Comment:
if you want to be very clever, since you know the format of this file you could use a struct with tags (https://www.digitalocean.com/community/tutorials/how-to-use-struct-tags-in-go)

Reply:
I was thinking about this and I think I'll follow that way especially because the "read version" feature will need to be extracted to a separated util (because it'll be used across different commands).